### DISCUSSION TITLE
Is caer.train_test_split() not part of the library anymore?

### DISCUSSION THREAD

**Discussion by lrakla** [jasmcaus/caer#77] [General]
On running, it shows module not found.

**Comment by jasmcaus** [comment]
`train_test_split` is a Scikit Learn function. The equivalent in `Caer` is [`train_val_split`. ](https://github.com/jasmcaus/caer/blob/11ff37867c2cb86a92aceb0ac24accb3607e3635/caer/core.py#L63)
